### PR TITLE
UPP-changes

### PR DESCRIPTION
**Pull request by KaiserBlackwood**
# About the pull request

This PR fixes the sorry state that UPP Gear and Uniform vendors were in, and gives them a selection of equipment on par with their USCM counterparts.

# Explain why it's good for the game

Makes UPP gear prep more streamlined and in general makes it more in line with standard round's gear prep. 
So a big QOL improvement for HVH.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
add: Added a standard choice of pouches
add: Added more point-buy equipment to the UPP vendor
add: Added UPP grenade packets
add: Added UPP grenade belts
add: Added UPP SL Weapon Kit
fix: Fixed prices in UPP vendors
qol: Streamlined gear/uniform vendors for UPP
spellcheck: renamed "Motion Detector" to "UPP Motion Detector" in all UPP Vendors

/:cl:

### CODE REVIEW

**Comment by harryob** [comment]
your changelog is sentence fragments, they need to be `add: Added more point-buy equipment to the UPP vendor.`

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

File: code/modules/gear_presets/upp.dm [hunk 1]
```diff
@@ -2830,6 +2884,10 @@
 
 /datum/equipment_preset/upp/officer/ley_gen
 	name = "UPP General Polkovnik (Cryo)"
+=======
```
Comment (Steelpoint) [inline code/modules/gear_presets/upp.dm:2887]: get rid of the ======

File: code/modules/gear_presets/upp.dm [hunk 2]
```diff
@@ -2830,6 +2884,10 @@
...
+/datum/equipment_preset/upp/officer/flag/ley_gen/dressed
+	name = "UPP General Polkovnik"
+>>>>>>> master
```
Comment (Steelpoint) [inline code/modules/gear_presets/upp.dm:2890]: get rid of the >>>>>> master

**Review by Steelpoint — COMMENTED** [review]
Forgot to remove a few lines from the merge conflict resolution.

**Pull request closed without merging**